tripal_chado/src/Commands/ChadoCheckTermsAgainstYaml.php

```diff
@@ -235,6 +236,123 @@ protected function chadoCheckTerms_findCVProblems(&$problems, &$solutions, &$sum
     }
   }
 
+  /**
+   * Checks for chado records lacking a type definition.
+   *
+   * @param array $problems
+   *   Array containing details for either errors or warnings
+   * @param array $solutions
+   *   Array containing possible solutions for either errors or warnings
+   * @param array $summary_rows
+   *   Infomation for the output table
+   * @param array $options
+   *   Options from drush command line
+   **/
+  protected function chadoCheckTerms_findBundleProblems(&$problems, &$solutions, &$summary_rows, $options) {
```

Comment:
I don't see anything here that excludes the null pub and null contact from being indicated as not having a bundle term and thus having one added as a solution. However the PR description/comments and Codefest convos implied that the null contact/pub would not be part of the generic bundle :confused: